### ISSUE TITLE
Register the a1_minimal_distributed_lm_001 training policy and run definition

### ISSUE THREAD

**Issue by AtlantisPleb** [OpenAgentsInc/openagents#4454]
## Objective

Add a canonical OpenAgents run definition for
`a1_minimal_distributed_lm_001`.

The run definition should use the existing
`ComputeTrainingRunDefinition`/training-policy metadata surface rather than a
parallel launch path.

## Scope

- Add or seed a training policy for `a1_minimal_distributed_lm_001`.
- Bind tokenizer digest, dataset digest, model config, optimizer config,
  aggregation rule, checkpoint family, validator policy, environment ref, and
  benchmark package refs.
- Ensure windows, manifests, TRN events, and closeouts can carry the run
  definition ref.
- Expose the run definition through the existing route:
  `GET /v1/kernel/compute/training/policies/{training_policy_ref}/run-definition`.

## Likely files

- `crates/openagents-kernel-core/src/compute.rs`
- `crates/openagents-kernel-core/src/pylon_training.rs`
- `apps/nexus-control/src/lib.rs`
- `apps/nexus-control/src/kernel.rs`
- `apps/nexus-control/src/training_trn_mapping.rs`
- `docs/plans/compute-training-run-definition-contract.md`

## Acceptance criteria

- The run definition is returned by the existing policy route.
- The definition includes the Psionic lane identity and fixed digest refs.
- Training windows for the run carry the correct work class, replica type,
  aggregation, checkpoint, and validator refs.
- Tests prove the run definition refuses missing tokenizer/dataset/checkpoint
  identity.

## Dependencies

- Psionic lane contract issue: https://github.com/OpenAgentsInc/psionic/issues/948
- Psionic tokenizer/dataset bundle issue: https://github.com/OpenAgentsInc/psionic/issues/949

## Source

Created from `docs/autopilot/2026-04-27-a1-minimal-distributed-lm-issue-backlog.md`, section 9: OpenAgents Run Definition.

**Comment by AtlantisPleb** [comment]
Implemented on main in commit `23254313a` (`Register A1 minimal distributed LM run definition`).

Summary:
- Added the canonical `a1_minimal_distributed_lm_001` run-definition metadata and OpenAgents policy refs, including fixed tokenizer, tokenized dataset, validation set, model, optimizer, scheduler, aggregation, checkpoint, validator, environment, and benchmark identities.
- Extended `ComputeTrainingRunDefinition`/metadata validation so A1 minimal definitions reject missing tokenizer, dataset, validation, model/optimizer/scheduler, aggregation, checkpoint, validator, environment, or benchmark identity.
- Projected the new fields through the existing `/v1/kernel/compute/training/policies/{training_policy_ref}/run-definition` route.
- Carried run-definition identity into training-window metadata, TRN window events, closeout metadata, and additive Pylon manifest `run_definition_ref`.
- Documented the concrete A1 minimal run definition in `docs/plans/compute-training-run-definition-contract.md`.

Validation:
- `rustfmt --edition 2024 --check crates/openagents-kernel-core/src/compute.rs crates/openagents-kernel-core/src/pylon_training.rs crates/psionic-train-contract/src/lib.rs apps/nexus-control/src/kernel.rs apps/nexus-control/src/lib.rs apps/nexus-control/src/training_trn_mapping.rs`
- `git diff --check`
- `cargo test -p openagents-kernel-core a1_minimal_distributed_lm_run_definition --lib`
- `cargo test -p openagents-kernel-core validates_training_registry_contracts --lib`
- `cargo test -p openagents-kernel-core pylon_training --lib`
- `cargo test -p psionic-train-contract`
- `cargo test -p nexus-control kernel_compute_training_run_definition_route_projects_a1_minimal_fixed_refs`
- `cargo test -p nexus-control kernel_compute_training_run_definition_route_projects_registered_contracts`

Note: `nexus-control` still emits pre-existing dead-code warnings in the focused test build; the targeted tests pass.
